_posts/2017-11-15-intro-html-css.md

```diff
@@ -318,7 +318,7 @@ To style our card divs, let's add these properties into our class selector in CS
 
 ```css
 .card-wrapper {
-  text-align: center; /* Keep our cards centered */
+  text-align: center; /* Keep text in our cards centered */
```

Comment:
Wait no this keeps our cards centered too (so our jank % margins don't look off-centered)

Reply:
whoops, my bad!